documentation/docs/60-appendix/15-debugging.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Breakpoint Debugging
```

Comment:
I'd make this a bit more generic for the lefthand menu. I'd also move the page down an item to come after "Integrations"
```suggestion
title: Debugging
```